### PR TITLE
Switch to namespaced Twig classes

### PR DESCRIPTION
**Pull request by derrabus**
Hello there. Following the news that using Twig's old PEAR-style classes [will trigger deprecation notices soon](https://symfony.com/blog/new-in-twig-namespaced-classes), I though it might be a good idea to switch this lovely codebase to the namespaced classes now.

### CODE REVIEW

**Comment by derrabus** [comment]
You're right. I've updated the composer.json file.

**Comment by derrabus** [comment]
CI failure is unrelated to my changes. The unit tests complain about the deprecated spaceless tag which is tackled in #300.

File: composer.json [hunk 1]
```diff
@@ -22,12 +22,15 @@
     "require":      {
         "php":  ">=5.6.0"
     },
+    "conflict": {
```
Comment (garak) [inline composer.json]: What is hte purpose of adding a `conflict` here? Aren't constraints in `require-dev` enough?

Comment (derrabus) [reply composer.json]: This prevents this library to be installed together with incompatible Twig releases. And no, `require-dev` is ignored if you consume this library as a dependency.

**Pull request merged**